files/build_templates/swss.service.j2

```diff
@@ -18,8 +18,10 @@ ExecStartPre=/usr/bin/docker exec database redis-cli -n 1 FLUSHDB
 ExecStartPre=/usr/bin/docker exec database redis-cli -n 2 FLUSHDB
 
 {% if sonic_asic_platform == 'mellanox' %}
-ExecStartPre=/etc/init.d/sxdkernel start
+TimeoutStartSec=3min
 ExecStartPre=/usr/bin/mst start
+ExecStartPre=/usr/bin/mlnx-fw-upgrade.sh
```

Comment:
No need to restart service. FW update procedure starts before SX kernel drivers and SDK.